src/compiler/transformers/es2015.ts

```diff
@@ -3337,8 +3337,12 @@ namespace ts {
         scoped: false,
         priority: 0,
         text: `
+            var __extendStatics = (this && this.__extendStatics) ||
+                Object.setPrototypeOf ||
+                ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
+                function (d, b) { for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p]; };

```

Comment:
If you are using `--importHelpers` then you can't override any imported helper in any case.